Donor/Profiles/DonorsProfile.cs

```diff
@@ -1,12 +1,20 @@
 using AutoMapper;
+using Donor.Models;
+
 
 namespace Donor.Profiles
 {
     public class DonorsProfile : Profile
     {
         public DonorsProfile()
         {
-            CreateMap<Models.DonorDto, Entities.Donor>();
+          CreateMap<DonorDto, Entities.Donor>()
+                .ForMember(dest => dest.DonationPreferences, opt => opt.Ignore());
```

Comment:
This mapping is going to change base on the relationship between Donor and Organ classes